lib/stock.js

```diff
@@ -8,8 +8,8 @@ module.exports = function (restClient) {
     }
     return false;
   }
-  module.check = (sku) => {
-    url += `check?sku=${sku}`;
+  module.check = (stock) => {
```

Comment:
we need to keep backward compatibility, can you please change it to `module.check = (sku, stockId) => {`?

Reply:
Yes I want to change it but its remains an object :)

```js
stockProxy.check({
      sku: req.params.sku,
      stockId: config.msi.enabled ? (req.params.stockId ? req.params.stockId : _getStockId(req.params.storeCode)) : null
    })
```

do you prefer to change this code ?